api/nakadi-event-bus-api.yaml

```diff
@@ -263,6 +271,11 @@ paths:
           description: Unprocessable Entity
           schema:
             $ref: '#/definitions/Problem'
+        '403':
+          description: Access forbidden
```

Comment:
It won't hurt, and it will be clearer to users. I will add it to all 403s that can be caused by event type authorization failures.